spec/rmt/ssl/new_ca_password_dialog_spec.rb

```diff
@@ -26,9 +26,10 @@
   describe '#initialize' do
     it 'creates the UI elements' do
       expect(dialog.instance_variable_get(:@dialog_heading)).to eq('Setting CA private key password')
-      expect(dialog.instance_variable_get(:@dialog_label)).to eq('Please set new CA private key password')
-      expect(dialog.instance_variable_get(:@password_field_label)).to eq('New CA private key &Password')
-      expect(dialog.instance_variable_get(:@password_confirmation_field_label)).to eq('New Password &Again')
+      expect(dialog.instance_variable_get(:@dialog_label)).to eq('Please set a password for the CA private key\n' \
```

Comment:
please use double quotes otherwise `\n` is not recognized in the string
![screenshot_20180703_125844](https://user-images.githubusercontent.com/892391/42216089-e2326e08-7ec0-11e8-8dd2-3cc79a966f43.png)